README.md

```diff
@@ -34,6 +34,14 @@ docker pull minio/mc:edge
 docker run minio/mc:edge ls play
 ```
 
+**Note:** These examples refer to the [_play_ environment](#test-your-setup). If you want to try the same commands against a different S3 compatible server, start the container this way:
```

Comment:
How about - Above examples run `mc` against Minio [_play_ environment](#test-your-setup) by default. To run `mc` against other S3 compatible servers, start the container this way: